ibazel/output_runner/output_runner_test.go

```diff
@@ -93,6 +93,8 @@ func TestMatchRegex(t *testing.T) {
 	buf.WriteString("buildozer 'add deps test_dep2' //target2:target2\n")
 	buf.WriteString("buildifier 'cmd_nvm' //target_nvm:target_nvm\n")
 	buf.WriteString("not_a_match 'nvm' //target_nvm:target_nvm\n")
+	// Duplicate match, to be deduplicated.
+	buf.WriteString("buildifier 'cmd_ignore' //target_ignore:target_ignore\n")
```

Comment:
Sorry to bring something like this up do late in the review process but I didn't realize till just now. Does this imply that a program who emits:

```
buildifier 'add deps test_dep1' //target
buildifier 'add deps test_dep2' //target
```

Would only have the first buildifier run? Wouldn't we want to dedupe based on both the command and arguments so that we don't have to do 2 builds in order to add 2 deps to a binary?

Reply:
This already dedupes based on command and arguments. This just looks unclear (in fact, when you raised this, I had to read back over it a few times to convince myself it's correct).

This is a `buildifier` call not `buildozer`, and the matcher for it does not use string parts in its call to `buildifier` below.

I've updated this with an extra string and some comments to make it clearer the diff between this line and the additional `buildozer` line I've added as well.

Reply:
Ah! I think I see it now. `buildifier` != `buildozer`. I think this invocation of `buildifier` is a bit strange. Wouldn't `buildifer` be applied to a single file to format it? Why is it taking a `cmd_ignore` and a target?

Reply:
lol, copy-pasta? I was copying line 94 (as another bazel output to match on) but changing it slightly, so that it'd match the regex but not fully match the line.

I can do something else with it.

Reply:
I mostly look at the test cases to see examples of how it would actually be used. My tiny monkey brain would be a little less taxed if the examples were a little more straight forward. Sorry!

Reply:
lol, np, I just updated this to match the `buildozer` and `gazelle` suggestions from documentation.